application/src/ext-test/java/org/opentripplanner/ext/empiricaldelay/internal/graphbuilder/TripDelaysMapperTest.java

```diff
@@ -0,0 +1,147 @@
+package org.opentripplanner.ext.empiricaldelay.internal.graphbuilder;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.ext.empiricaldelay.internal.model.TripDelaysDto;
+import org.opentripplanner.ext.empiricaldelay.model.EmpiricalDelay;
+import org.opentripplanner.graph_builder.issue.api.DataImportIssueStore;
+import org.opentripplanner.graph_builder.issue.service.DefaultDataImportIssueStore;
+import org.opentripplanner.transit.model.framework.Deduplicator;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+class TripDelaysMapperTest {
+
+  private static final String FEED_ID = "F";
+  private static final String WEEKEND = "Weekend";
+  private static final String MON_FRI = "Mon-Fri";
+  private static final FeedScopedId TRIP_ID_A = new FeedScopedId(FEED_ID, "Trip-A");
+  private static final FeedScopedId TRIP_ID_OTHER = new FeedScopedId(FEED_ID, "Trip-OTHER");
+  private static final FeedScopedId STOP_ID_A = new FeedScopedId(FEED_ID, "Stop-A");
+  private static final FeedScopedId STOP_ID_B = new FeedScopedId(FEED_ID, "Stop-B");
+  private static final FeedScopedId STOP_ID_C = new FeedScopedId(FEED_ID, "Stop-C");
+  private static final FeedScopedId STOP_ID_D = new FeedScopedId(FEED_ID, "Stop-D");
+  private static final FeedScopedId STOP_ID_E = new FeedScopedId(FEED_ID, "Stop-E");
+  private static final Map<FeedScopedId, List<FeedScopedId>> STOP_PATTERNS = Map.of(
+    TRIP_ID_A,
+    List.of(STOP_ID_A, STOP_ID_B, STOP_ID_C)
+  );
+  private static final EmpiricalDelay EMPIRICAL_DELAY_A = new EmpiricalDelay(19, 73);
+  private static final EmpiricalDelay EMPIRICAL_DELAY_B = new EmpiricalDelay(15, 63);
+  private static final EmpiricalDelay EMPIRICAL_DELAY_C = new EmpiricalDelay(12, 53);
+  private static final EmpiricalDelay EMPIRICAL_DELAY_SAME_AS_A = new EmpiricalDelay(19, 73);
+
+  private final Deduplicator deduplicator = new Deduplicator();
+  private final DataImportIssueStore issueStore = new DefaultDataImportIssueStore();
+  private final TripDelaysMapper subject = new TripDelaysMapper(
+    STOP_PATTERNS,
+    issueStore,
+    deduplicator
+  );
+
+  @Test
+  void testMap() {
+    var dto = new TripDelaysDto(TRIP_ID_A);
```

Comment:
Is using `Dto` as a suffix a pattern we want to encourage?

Reply:
We we hava da DTO, then I think it si ok. But, in this case I think the name is wrong. I will rename it to `TripDelaysAgregator`